vpdq/python/vpdq.pyx

```diff
@@ -137,27 +161,30 @@ def computeHash(
     if downsample_height < 0:
         raise ValueError("Downsample_height must be non-negative")
     cdef vector[vpdqFeature] vpdq_hash;
-    vid = cv2.VideoCapture(str_path)
-    frames_per_sec = vid.get(cv2.CAP_PROP_FPS)
-    if downsample_width == 0:
-        downsample_width = vid.get(cv2.CAP_PROP_FRAME_WIDTH)
+    
+    width, height, frames_per_sec = read_video_stream_info(
+        str_path,
+        "vpdqPY",
+    )
 
-    if downsample_height == 0:
-        downsample_height = vid.get(cv2.CAP_PROP_FRAME_HEIGHT)
+    if downsample_width > 0:
+        width = downsample_width
     
```

Comment:
Inverted logic because width is retrieved from `read_video_stream_info`() regardless if downsamples are 0. Also the original was not really proper naming since

BTW I think read_video_stream_info() should probably just be a function in `filehasher.cpp` that is called by `hashVideoFile()` because there's no need for the programmer to have to get these attributes. It would reduce complexity.

Reply:
I open to these changes if you want to make them! If you aren't sure, feel free to leave a comment to that effect to prompt someone in the future.